packages/edit-post/src/components/header/header-toolbar/index.js

```diff
@@ -96,18 +140,25 @@ function HeaderToolbar() {
 					'Add block',
 					'Generic label for block inserter button'
 				) }
+				showTooltip={ ! showIconLabels }
 			/>
-			{ isLargeViewport && <ToolbarItem as={ ToolSelector } /> }
-			<ToolbarItem as={ EditorHistoryUndo } />
-			<ToolbarItem as={ EditorHistoryRedo } />
-			<ToolbarItem
-				as={ TableOfContents }
-				hasOutlineItemsDisabled={ isTextModeEnabled }
-			/>
-			<ToolbarItem
-				as={ BlockNavigationDropdown }
-				isDisabled={ isTextModeEnabled }
-			/>
+			{ ( isWideViewport || ! showIconLabels ) && overflowItems }
+			{ ! isWideViewport && ! isSmallViewport && showIconLabels && (
+				<Dropdown
+					contentClassName="edit-post-header__dropdown"
+					position="bottom right"
+					renderToggle={ ( { isOpen, onToggle } ) => (
+						<Button
+							className="button-toggle"
+							aria-expanded={ isOpen }
+							icon={ chevronDown }
```

Comment:
I know there was some discussion on the PR about the design of this, but should this also be a text button?

It seems inconsistent to have an icon button in this mode.

It also looks like it needs a label.

Reply:
This raises the question of what label to give this button. There are two of them, and just calling them both "More" seems a bit confusing, since there's no clear answer to "More what?" ...which I guess just loops back to the same issue we keep running into: the top toolbar is not organized... or rather, what organization it does have is very vague and arbitrary.

Reply:
Hmm, yep, Settings and Options are also often interchangeable, but they are in the same menu at small breakpoints with not much to distinguish them.

I do think there's a bit of a separation in the way options are grouped. The items on the left are more about interacting with content, while those on the right are more related to the editor features or publishing. Not sure I'm smart enough to dilute that into button labels though. The best I can come up with is:
- 'More content tools'
- 'More editor options'

Reply:
Yeah, we seem to be going around in circles on this. "More tools and options" has already become "Options", and these dropdowns include that and a heap of other stuff that isn't easy to label as a group 😬

Reply:
> The best I can come up with is:
> 'More content tools'
> 'More editor options'

That's much better than my 0 ideas so far 😅 
We'd need them to be shorter though. And clicking on an "options" button only to find another "options" button inside would be a bit 😭. 
Putting the sidebar toggle inside the options menu doesn't seem right either. I wonder if we could leave it out and instead pop the "switch to draft" in the "options"?

Reply:
Perhaps instead of having essentially a more menu inside of another more menu on the right, just have one more menu that stuff moves into as necessary (much like how the overflow menu in the header of the Twenty Nineteen theme works). The toolbar should also be reorganized so the most important controls are listed first, so they remain visible at more viewport widths, e.g. "Publish" should come before "Save draft" and "Preview".